content/source/docs/enterprise/user-accounts/access.html.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: "enterprise"
+page_title: "Access - Accounts - Terraform Enterprise"
+sidebar_current: "docs-enterprise-accounts-access"
+description: |-
+  Terraform Enterprise allows collaboration on resources. However, there are several levels of access.
+---
+
+# Access
+
+Terraform Enterprise allows collaboration on resources. However, there are several access levels available for users.
+
+### Read
+
+Read is the lowest level of access within Terraform Enterprise. Users with read access on a resource can do just that. They are able to view the resource, but are unable to perform any CRUD actions.
```

Comment:
- "Users with read access on a resource can view it, but are unable to modify or delete it."